updated quickstart examples with some GIE related changes

nirrozenbaum opened the pull request:
This PR handles few minor changes in quickstart examples:
- `kv-cache-scorer` was updated in GIE to `kv-cache-utilization-scorer`.
- in some of the examples, a documentation specifying one can use either llm-d scheduler image OR GIE image is similar. in most of the cases that's not correct and we want to avoid the confusion of users keep asking us about the differences.
in llm-d we should always use llm-d images unless there is a good reason not to do so. (e.g., if in the future everything will be consolidated into GIE).
- GIE doesn't contain `plugins-v2.yaml` anymore. updated that section to point to `default-plugins.yaml`.


cc: @Gregory-Pereira @vMaroon

vMaroon commented on quickstart/examples/precise-prefix-cache-aware/gaie-kv-events/values.yaml:
```diff
@@ -49,15 +41,15 @@ inferenceExtension:
                 enableMetrics: true                   # enable kv-block index metrics (prometheus)
                 metricsLoggingInterval: 60000000000   # log kv-block metrics as well (1m in nanoseconds)
-        - type: kv-cache-scorer # kv-cache-utilization
+        - type: kv-cache-utilization-scorer
```
AFAIK this wasn't in v0.2.1, is it?

nirrozenbaum replied:
right. 
this is the current state in GIE main, which will result in a first RC in few days.
this PR tries to minimize the disruption and prepare for that.
(main should be a moving target, not pinned to an old GIE version).

nirrozenbaum replied:
building on main to run examples is not so great.
it requires doing all the changes only after cutting a first GIE RC, this pattern is not very recommended 😃.

nirrozenbaum replied:
created a new issue to track this #166

nirrozenbaum replied:
nothing that seems related to the chart.
having said that, we do need to update the GIE inferencepool charts when llm-d scheduler releases.

nirrozenbaum commented:
> @nirrozenbaum Just confirming that this does not require using the new `inference-scheduler` RC? If so we should update the image tags in this PR, if not happy to merge immedieately

I updated the PR to use latest RC and updated the target port field accordingly.
This change is for the uncoming release, so it's better to update scheduler used RC
